contracts/cw20-base/src/enumerable.rs

```diff
@@ -16,7 +16,7 @@ pub fn query_all_allowances(
 ) -> StdResult<AllAllowancesResponse> {
     let owner_addr = deps.api.addr_validate(&owner)?;
     let limit = limit.unwrap_or(DEFAULT_LIMIT).min(MAX_LIMIT) as usize;
-    let start = start_after.map(Bound::exclusive);
+    let start = start_after.map(|s| Bound::ExclusiveRaw(s.as_bytes().into()));
```

Comment:
This makes a clone I think.

Doesn't [`s.into_bytes()`](https://doc.rust-lang.org/stable/std/string/struct.String.html#method.into_bytes) do the same without a clone?